print-bgp.c

```diff
@@ -2654,6 +2673,16 @@ bgp_attr_print(netdissect_options *ndo,
         }
         break;
     }
+    case BGPTYPE_OTC:
+    {
+        if (len < 4) {
+            ND_PRINT("invalid len");
+            break;
+        }
+        ND_TCHECK_LEN(pptr, len);
```

Comment:
You don't need the TCHECK since you're using GET_ to get the U4 below.

Reply:
Thanks, fixed.